src/core/loss_detection.c

```diff
@@ -1153,12 +1156,15 @@ QuicLossDetectionDiscardPackets(
 
     if (AckedRetransmittableBytes > 0) {
         const QUIC_PATH* Path = &Connection->Paths[0]; // TODO - Correct?
-        if (QuicCongestionControlOnDataAcknowledged(
-                &Connection->CongestionControl,
-                TimeNow,
-                LossDetection->LargestAck,
-                AckedRetransmittableBytes,
-                Path->SmoothedRtt)) {
+
+        QUIC_ACK_EVENT AckEvent = CreateQuicAckEvent();
+        AckEvent.TimeNow = TimeNow;
+        AckEvent.LargestPacketNumberAcked = LossDetection->LargestAck;
+        AckEvent.NumRetransmittableBytes = AckedRetransmittableBytes;
+        AckEvent.SmoothedRtt = Path->SmoothedRtt;
+        AckEvent.IsImplicit = TRUE;
```

Comment:
`IsImplicit` isn't being used right now, right? Is this for some future CC? I'd rather not add things that are currently unnecessary, and instead delay adding them until they are required,

Reply:
Agreed.